### DISCUSSION TITLE
[Feature] Resize images in feeds to a maximum with CSS

### DISCUSSION THREAD

**Discussion by jan-vandenberg** [FreshRSS/FreshRSS#6563] [Q&A] [answered]
**Is your feature request related to a problem? Please describe.**
Images in RSS feeds usually take up a lot of my screen. Blogs/sites usually scale down images based on screen size (pc/mobile etc.), but they put the full-blown images in the RSS feeds.

**Describe the solution you’d like**
I want all images in all feeds in FreshRSS to be a maximum size. I (think) this can be done in a template with some CSS?

**Describe alternatives you’ve considered**
I looked at templates that do this, but found none.

**Comment by Frenzie** [comment]
max-width: something 

 on

 .flux_content img

or something like that (sorry, on my phone)

**Comment by math-GH** [comment]
Have you tried another `Content width`?

![grafik](https://github.com/FreshRSS/FreshRSS/assets/1645099/f93e3fe6-9e0c-4b22-87b0-c000494cd30c)

**Comment by jan-vandenberg** [comment]
Yes I have tried this, it resizes the images a bit more but not to a fixed max width.

**Comment by math-GH** [comment, marked as answer]
With the CustomCSS extension:

```
.flux_content .content .text img, 
.flux_content .content .text picture {
  max-width: 300px;
}
```

Does it help you?

**Comment by jan-vandenberg** [comment]
Awesome this works!
